chsdi/static/doc/source/conf.py

```diff
@@ -176,7 +177,7 @@
 #html_show_sourcelink = True
 
 # If true, "Created using Sphinx" is shown in the HTML footer. Default is True.
-#html_show_sphinx = True
+html_show_sphinx = True
```

Comment:
This is intended?

Reply:
yes, it changes nothing. but it is more explicite